bbc/simorgh
# WSTEAM1-700: Display media on Live Page

Resolves JIRA [WSTEAM1-700]

Overall changes
======
Adds support for rendering video on a Live Page post

Code changes
======

- Adds a video handler to the PostContent `componentsToRender` mapping.
- Adds a LivePageMediaPlayer container to render video on the Live Page.
- Adds modules to retrieve AV data from blocks for Optimo, CPS and Live Page blocks.

Testing
======
1. Setup the Next.js app as per instructions [here](https://github.com/bbc/simorgh/tree/latest/ws-nextjs-app#setup
).
2. Visit http://localhost.bbc.co.uk:7081/pidgin/live/c7p765ynk9qt?renderer_env=test
3. Posts containing video should display the media player.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
